bundles/org.openhab.binding.enocean/README.md

```diff
@@ -38,7 +38,7 @@ This binding is developed on and tested with the following devices
 - USB300 and EnOceanPi gateways
 - The following Eltako actuators:
   - FSR14 (light switch)
-  - FSB14 (rollershutter)
+  - FSB14/61/71, FJ62 (rollershutter)
```

Comment:
No i have it tested only with an FJ62NP-230V but the technical specification of Eltako describes that FSB61 and FSB71 this devices uses the same telegrams.
ORG=0x05 --> 1 byte RPS confirmation telegram
ORG=0x07 --> 4 byte 4BS confirmation telegram

see:
Technical data of the wireless actuators, teach-in list, operating distances and contents of Eltako Wireless telegrams
[https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_highRes.pdf](https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_highRes.pdf)
Page T-16
CONFIRMATION TELEGRAMS OF BIDIRECTIONAL ACTUATORS.

**_FSB61NP-230V_**, **_FSB71_**, FJ62/12-36V DC, FJ62NP-230V

```
ORG=         0x05
Data_byte3 = 0x70 = upper stop position,
             0x50 = lower stop position,
             0x01 = Start up, 
             0x02 = Start down

If the actuator is stopped before the end of RV, only the actual elapsed time
is sent indicating the direction in a ORG7 message with the same ID! This is
also the info that the engine has stopped now.

ORG =        0x07
Data_byte3 = driving time in 100 ms MSB
Data_byte2 = driving time in 100 ms LSB
Data_byte1 = 0x01 = driven up or 0x02 = driven down
Data_byte0 = 0x0A (not blocked) 
             or 
             0x0E (blocked)

Remark: The RV time must be set on the device so that the end position is
always reached. If the roller shutter is already at an end position, the relay is
switched on receipt of a drive command anyway (0x01 or 0x02 is sent) and it is
switched off on expiry of the RV. (0x70 or 0x50 is sent).

```

Reply:
I think you should not add '61/71' here since you didn't test it.